README.md

```diff
@@ -125,6 +125,9 @@ sendTo("enigma2.0", "send", {
     Placeholder for the next version (at the beginning of the line):
     ### **WORK IN PROGRESS**
 -->
+### WORK IN PROGRESS
```

Comment:
Please really use 

### **WORK IN PROGRESS**

(with the stars) and please ideally write human readable user direct english description of the changes in short ... links (I know also done in last changelog entry) are not that good for the users and display (sorry to bother with such formal stuff)

Reply:
fixed it

Reply:
Resulting changes look good